imports/ui/explorer-components/filter-row.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export class FilterRow extends Component {
+  render () {
+    const { filterStatus, myInvolvement, handleAssignedClicked, handleStatusClicked, filterType } = this.props
+    return (
+      <div className='flex pl3 pv3 bb b--very-light-gray bg-white'>
+        <div
+          onClick={() => handleStatusClicked(true)}
+          className={'f6 fw5 ph2 ' + (filterStatus ? 'mid-gray' : 'silver')}
+        >
+          {filterType[0]}
+        </div>
+        <div
+          onClick={() => handleStatusClicked(false)}
+          className={'f6 fw5 ml4 ph2 ' + (!filterStatus ? 'mid-gray' : 'silver')}
+        >
+          {filterType[1]}
+        </div>
+        <div
+          onClick={() => handleAssignedClicked()}
+          className={'f6 fw5 ml4 ph2 ' + (myInvolvement ? 'mid-gray' : 'silver')}
+        >
+          {filterType[2]}
+        </div>
+      </div>
+    )
+  }
+}
+
+FilterRow.propTypes = {
+  filterStatus: PropTypes.bool,
+  myInvolvement: PropTypes.bool,
+  filterType: PropTypes.array,
```

Comment:
I think this should be renamed to "filterNames". It makes more sense to name array properties and vars in general in the plural tense. The "type" -> "name" change makes more sense as it's the way the filters are named or labeled, it doesn't change their type really. "filterLabels" will be a good name too